### PR TITLE
Update codegen pixi task

### PR DESCRIPTION
**Pull request by Hofer-Julian**
- Running codegen again leads to a different result
- Linting is not needed
- Initializing Julia is needed

### CODE REVIEW

**Review by visr — APPROVED** [review]
Thanks. Yes I didn't run codegen after renaming nodes last week.

File: pixi.toml [hunk 1]
```diff
@@ -93,7 +93,7 @@ generate-testmodels = "python utils/generate-testmodels.py"
 tests = { depends_on = ["lint", "test-ribasim-python", "test-ribasim-core"] }
 # Codegen
 generate-python = { cmd = "julia --project utils/gen_python.jl && ruff format python/ribasim/ribasim/schemas.py"}
-codegen = { depends_on = ["generate-python", "lint"] }
+codegen = { depends_on = ["initialize-julia", "generate-python"] }
```
Comment (deltamarnix) [inline pixi.toml:96]: Linting was needed, because it solves the whitespacing issues. pre-commit didn't let me pass it.

Comment (Hofer-Julian) [reply pixi.toml:96]: Mmmh good to keep in mind.

Pre-commit runs anyway before, well, a commit, so I think we should still be fine :)

Comment (deltamarnix) [reply pixi.toml:96]: Correct, but that would mean you can never commit at once.

Comment (Hofer-Julian) [reply pixi.toml:96]: > Correct, but that would mean you can never commit at once.

That would be annoying. I can at least tell that I was able to commit at once.
If that was an outlier and most of the time pre-commit complains, then let's add the task again.

**Pull request merged**